remove-exec-depend/action.yaml

```diff
@@ -4,7 +4,25 @@ description: ""
 runs:
   using: composite
   steps:
+    - name: Check out default_branch
+      uses: actions/checkout@v3
```

Comment:
You can't add checkout here because the user might have checked out the repository already. This will break the user's checkout.

Reply:
I got it. Do you have any ideas to avoid it?

Reply:
Unfortunately, no. Probably you have to edit the caller-workflow (build-and-test.yaml) side.

Reply:
Or, it's good to fix `colcon` and remove this action itself.

Reply:
OK. I'll find some workaround.